resources/test/fixtures/U012.py

```diff
@@ -0,0 +1,23 @@
+"foo".encode("utf-8") # b"foo"
+"foo".encode("u8") # b"foo"
+"foo".encode() # b"foo"
+
+string = "hello there"
+string.encode("utf-8") # string.encode()
+
+"""
+Lorem
+
+Ipsum
+""".encode("utf-8") # b"\nLorem\n\nIpsum\n"
```

Comment:
pyupgrade would preserve the triple quotes with `b"""…"""`, which seems nicer.

Reply:
Fixed with below comment